compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -351,7 +351,10 @@ std::string keySetSerialize(concretelang::clientlib::KeySet &keySet) {
 concretelang::clientlib::SharedScalarOrTensorData
 valueUnserialize(const std::string &buffer) {
   auto inner = TransportValue();
-  if (inner.readBinaryFromString(buffer).has_failure()) {
+  if (inner
+          .readBinaryFromString(buffer,
+                                mlir::concretelang::python::DESER_OPTIONS)
+          .has_failure()) {
     throw std::runtime_error("Failed to deserialize Value");
```

Comment:
Sorry I saw that after the merge, but here the error message should be forwarded.